cypress/integration/settings-spec.js

```diff
@@ -31,8 +53,26 @@ describe('settings page', () => {
   })
 
   it('only the first element should be selected on load', () => {
-    firstElementOfTheSettingsMenuList.should('have.attr', 'nav-selected', 'true')
+    cy.get('.list').children().first().should('have.attr', 'nav-selected', 'true')
 
     firstElementOfTheSettingsMenuList.next().should('have.attr', 'nav-selected', 'false')
   })
+
+  it('language of the app should change', () => {
+    cy.enter()
+    languageSettingsPage.popupTitleElement().should('have.text', enJson['language-setting'])
+
+    languageSettingsPage.popupTextElement().should('have.text', languageSettingsPopupEnglishText)
+    cy.enter()
+
+    cy.get('.left').should('have.text', 'Search')
```

Comment:
'Search' here use `enJson['softkey-search']`

Reply:
consider making Softkey also in the page-object, then you can `softkey.leftText()` or just `.left()`

Reply:
fixed